addons/hr_holidays/models/hr_leave_accrual_plan_level.py

```diff
@@ -16,14 +16,8 @@
 MONTHS = ['jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'oct', 'nov', 'dec']
 # Used for displaying the days and reversing selection -> integer
 DAY_SELECT_VALUES = [str(i) for i in range(1, 29)] + ['last']
-
-def _get_selection_days(self):
-    lang = get_lang(self.env).code
-    return [
-        (DAY_SELECT_VALUES[i - 1],
-        num2words(i, lang=lang, to='ordinal_num') if i < 29 else _('last day'))
-        for i in range(1, 30)
-    ]
+DAY_SELECT_DISPLAY = [str(i) for i in range(1, 29)] + ['last day']
+DAY_SELECT_SELECTION = list(zip(DAY_SELECT_VALUES, DAY_SELECT_DISPLAY))
```

Comment:
If the purpose of the fix is to remove `num2words`, you can just do this to keep it simpler:

```suggestion
def _get_selection_days(self):
    return [
        (DAY_SELECT_VALUES[i - 1],
        str(i) if i < 29 else _('last day'))
        for i in range(1, 30)
    ]
```

You can also remove the import of `num2words`

Reply:
But isn't it the same just faster ?
(I should still change list to tuple to ensure it's immutable)

Reply:
and maybe add a function that return if it's really better not to touch the rest

Reply:
Yes, I think it's better to keep the function.